source/pdo_sqlsrv/pdo_dbh.cpp

```diff
@@ -1204,81 +1219,79 @@ char * pdo_sqlsrv_dbh_last_id( pdo_dbh_t *dbh, const char *name, _Out_ size_t* l
 int pdo_sqlsrv_dbh_quote( pdo_dbh_t* dbh, const char* unquoted, size_t unquoted_len, char **quoted, size_t* quoted_len,
                           enum pdo_param_type /*paramtype*/ TSRMLS_DC )
 {
-	PDO_RESET_DBH_ERROR;
-	PDO_VALIDATE_CONN;
+    PDO_RESET_DBH_ERROR;
+    PDO_VALIDATE_CONN;
     PDO_LOG_DBH_ENTRY;
 
-	pdo_sqlsrv_dbh* driver_dbh = reinterpret_cast<pdo_sqlsrv_dbh*>( dbh->driver_data );
```

Comment:
I think these are all whitespace changes actually. I changed to spaces to make consistent indenting easier. The old and new code are identical except for whitespace.